CHANGELOG.md

```diff
@@ -19,13 +19,21 @@ A breaking change will get clearly marked in this log.
   - There's a new field `trade_type` that can be either `orderbook` or `liquidity_pool`. You can filter by that field.
   - Liquidity pool trades will contain the field `liquidity_pool_fee_bp` and either `base_liquidity_pool_id` or `counter_liquidity_pool_id`.
   - There are a few breaking changes to this endpoint listed in the section below.
+- A new kind of trustline called `liquidity_pool_shares` was added and clients can now filter the results from the `/accounts` endpoint by `liquidity_pool`. This will include a new kind of trustline in the account `balances` array, causing a breaking change as described in the section below ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)).
 
 ### Breaking changes
 
 - The response from `/trades` endpoint can now contain two different kinds of trades, `orderbook` and `liquidity_pool` (new), which brought a few breaking changes ([#685](https://github.com/stellar/js-stellar-sdk/pull/685)):
   - Some previously mandatory fields were made optional: `counter_offer_id`, `base_offer_id` will only show up in orderbook trades while only one of `base_account` and `counter_account` will appear in liquidity pool trades.
   - The `price` field changed from `{n: number; d: number;}` to `{n: string; d: string;}`.
   - The links to "base" and "counter" returned from horizon can now point to either an account or a liquidity pool.
+- The `balances` array from an account response now supports liquidity pool balances ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)).
+  - the `asset_type` field can now be `liquidity_pool_shares`.
+  - `buying_liabilities`, `selling_liabilities`, `asset_code`, and `asset_issuer` are omited from the response for pool shares because they are not relevant to liquidity pools.
```

Comment:
typo nit: omitted
```suggestion
  - `buying_liabilities`, `selling_liabilities`, `asset_code`, and `asset_issuer` are omitted from the response for pool shares because they are not relevant to liquidity pools.
```

Reply:
Addressed in https://github.com/stellar/js-stellar-sdk/pull/688/commits/8ec0cdc93f295652cade3d12f135d4dec0695752.